### ISSUE TITLE
Conflict dialogue almost never gets the icons right...

### ISSUE THREAD

**Issue by oparoz** [owncloud/core#23070]
### Steps to reproduce
1. Upload a log file. The row shows the plain grey icon
2. Upload it again

You can try with an Excel file as well if you prefer colours
### Expected behaviour

The conflict dialogue should have the same image or icon for both sides
### Actual behaviour

The conflict dialogue doesn't have a preview for the existing file, on the right.

This URL is 404
`index.php/core/preview.png?file=%2F%2FThat+One+Privacy+Guy%27s+VPN+Comparison+Chart.xlsx&x=96&y=96&c=5a623410f086abec39c2ff057863cfdc&forceIcon=0`

From the config below, you can see that I have font preview enabled. If I repeat the steps with a font, I get a different problem:
- Row shows font preview
- CD left shows the plain grey icon
- CD right shows the preview

With a woff file, which is unsupported as far as previews are concerned, but which is recognized as a graphics file, instead of a font file (old issue)
- Row shows an image icon
- CD left shows the plain grey icon
- CD right is empty

So it seems there are different ways to get a preview and the conflict dialogue is using an outdated method.
It does work well for images though.

I've tested on 8.2 and got similar results for the log and Excel files.
### Server configuration

**ownCloud version:** stable9 owncloud@bd144efeb1c15a9373adc6b6724152df5aab6407

**The content of config/config.php:**

```
  'forcessl' => true,
  'loglevel' => '0',
  'memcache.local' => '\\OC\\Memcache\\Redis',
  'memcache.locking' => '\\OC\\Memcache\\Redis',
  'preview_max_x' => 756,
  'preview_max_y' => 756,
  'preview_max_scale_factor' => 2,
  'theme' => '',
  'maintenance' => false,
  'skeletondirectory' => '',
  'appstore.experimental.enabled' => true,
  'debug' => true,
  'enabledPreviewProviders' =>
  array (
    0 => 'OC\\Preview\\JPEG',
    1 => 'OC\\Preview\\PNG',
    2 => 'OC\\Preview\\GIF',
    3 => 'OC\\Preview\\Postscript',
    4 => 'OC\\Preview\\Font',
  ),
  'singleuser' => false,
```

**Are you using encryption:** no
#### ownCloud log (data/owncloud.log)

```
Nothing
```
#### Browser log

```
Nothing
```

---

@PVince81 @rullzer @MorrisJobke

**Comment by MorrisJobke** [comment]
cc @butonic

**Comment by PVince81** [comment]
@georgehrke

**Comment by ownclouders** [comment]
Hey, this issue has been closed because the label `status/STALE` is set and there were no updates for 7 days. Feel free to reopen this issue if you deem it appropriate.

(This is an automated comment from [GitMate.io](https://gitmate.io).)

**Comment by PVince81** [comment]
Please reopen if still an issue with 10.0.5 RC3.

@VicDeo

**Comment by VicDeo** [comment]
duplicate of https://github.com/owncloud/core/issues/30089 ?
Not fixed for 10.0.5, backport is pendng https://github.com/owncloud/core/pull/30125
